libcontainer/criu_linux.go

```diff
@@ -186,7 +186,7 @@ func criuNsToKey(t configs.NamespaceType) string {
 
 func (c *Container) handleCheckpointingExternalNamespaces(rpcOpts *criurpc.CriuOpts, t configs.NamespaceType) error {
 	if !c.criuSupportsExtNS(t) {
-		return nil
+		return fmt.Errorf("criu version does not meet the requirements")
```

Comment:
Maybe something more specific, like
```go
return fmt.Errorf("criu lacks support for %s namespace (old criu version?)", t)
```

Reply:
Yes, that's better. I will change it later.